### DISCUSSION TITLE
using another test's gold file

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#16253] [Q&A Tools] [answered]
What's the "right" way to have two (or more) tests share a gold file? The tests could all be in the same directory and therefore have different input and output filenames. I've been using soft (symbolic) links to make gold files for the other output files names that link to the first but that seems hacky and I'm not sure if that plays nicely with git.

**Comment by loganharbour** [comment, marked as answer]
They can just have the same output name.

We now serialize _all_ tests within the same directory to ensure that there are no race conditions.

**Reply by jessecarterMOOSE** [reply]
So are you suggesting changing `Outputs/file_base` in all the input files to the same string?

**Reply by aeslaughter** [reply]
Yes, it is typical to do this in the cli_args of the test specification

**Reply by jessecarterMOOSE** [reply]
:+1:
